CrewChiefV4/ACS/ACSGameStateMapper.cs

```diff
@@ -232,17 +234,34 @@ public GameStateData mapToGameStateData(Object memoryMappedFileStruct, GameState
             bool isInPits = shared.acsGraphic.isInPit == 1;
             int lapsCompleated = shared.acsGraphic.completedLaps;
             int numberOfSectorsOnTrack = shared.acsStatic.sectorCount;
- 
+            
             int numberOfLapsInSession = (int)shared.acsGraphic.numberOfLaps;
+            
+            float gameSessionTimeLeft = 0.0f;
+            if (!Double.IsInfinity(shared.acsGraphic.sessionTimeLeft))
```

Comment:
Very weird to see that in the C# code - means that AC has managed to put NaN or Infinity into the shared memory object